roles/ufw/tasks/main.yml

```diff
@@ -0,0 +1,33 @@
+- name: Install ufw
+  apt:
+      pkg: ufw
+      update-cache: yes
+      cache_valid_time: 3600
+
+- name: Allow everything and enable UFW
+  ufw:
+    state: enabled
+    policy: allow
+
+- name: Deny incomming policy
+  ufw:
+    direction: incoming
+    policy: deny
+
+- ufw:
+    rule: allow
+    name: OpenSSH
+
+- ufw:
+    rule: limit
+    port: ssh
+    proto: tcp
+
+- name: Allow access to http ports
+  ufw:
+    rule: allow
+    port: "{{ item }}"
+    proto: tcp
+  with_items:
```

Comment:
This could be `loop` now, right?

Reply:
Very true. :+1:  
Thanks, I have not seen your comment before the review, therefore I criticized the misspelling too.